packages/grid/_modules_/grid/components/DefaultFooter.tsx

```diff
@@ -1,43 +1,48 @@
 import * as React from 'react';
 import { useGridSelector } from '../hooks/features/core/useGridSelector';
+import { paginationSelector } from '../hooks/features/pagination/paginationSelector';
 import { rowCountSelector } from '../hooks/features/rows/rowsSelector';
 import { selectedRowsCountSelector } from '../hooks/features/selection/selectionSelector';
-import { optionsSelector } from '../hooks/utils/useOptionsProp';
+import { optionsSelector } from '../hooks/utils/optionsSelector';
 import { ApiContext } from './api-context';
 import { RowCount } from './RowCount';
 import { SelectedRowCount } from './SelectedRowCount';
-import { GridFooter } from './styled-wrappers/GridFooter';
+import { GridFooter } from './containers/GridFooter';
 
 export interface DefaultFooterProps {
-  paginationComponent: React.ReactNode;
+  PaginationComponent?: React.ReactNode;
 }
 
-export const DefaultFooter = React.forwardRef<HTMLDivElement, DefaultFooterProps>(
-  function DefaultFooter(props, ref) {
-    const { paginationComponent } = props;
-    const apiRef = React.useContext(ApiContext);
-    const totalRowCount = useGridSelector(apiRef, rowCountSelector);
-    const options = useGridSelector(apiRef, optionsSelector);
-    const selectedRowCount = useGridSelector(apiRef, selectedRowsCountSelector);
+export function DefaultFooter(props: DefaultFooterProps) {
+  const apiRef = React.useContext(ApiContext);
+  const totalRowCount = useGridSelector(apiRef, rowCountSelector);
+  const options = useGridSelector(apiRef, optionsSelector);
+  const selectedRowCount = useGridSelector(apiRef, selectedRowsCountSelector);
+  const pagination = useGridSelector(apiRef, paginationSelector);
 
-    const showSelectedRowCount =
-      !options.hideFooterSelectedRowCount && selectedRowCount > 0 ? (
-        <SelectedRowCount selectedRowCount={selectedRowCount} />
-      ) : (
-        <div />
-      );
+  const SelectedRowCountComponent =
```

Comment:
It's not a component:
```suggestion
  const selectedRowCount =
```

x3